source/Ship.cpp

```diff
@@ -4401,9 +4401,19 @@ double Ship::CalculateDeterrence() const
 			const Outfit *weapon = hardpoint.GetOutfit();
 			if(weapon->Ammo() && weapon->AmmoUsage() && !OutfitCount(weapon->Ammo()))
 				continue;
-			double strength = weapon->ShieldDamage() + weapon->HullDamage()
-				+ (weapon->RelativeShieldDamage() * attributes.Get("shields"))
-				+ (weapon->RelativeHullDamage() * attributes.Get("hull"));
+			// Ignore disabled and asteroid damages because they don't apply here.
+			// Ignore fuel and slowing damage because they don't affect combat ability.
+			// Over time effects are per frame but dont affect instantaneously so only multiply by 30.
+			// Burning and ion do not affect ships as much, but ion does so more than heat.
```

Comment:
I'm not sure what this sentence is saying.

Reply:
ion's worse than heat because the relative effect of 1 ion on a ship is worse than 1 heat for pirates in general?